graph_def_editor/transform.py

```diff
@@ -112,7 +114,14 @@ def assign_renamed_collections_handler(info, elem, elem_):
       transformed_name = name
     else:
       transformed_name = info.new_name(name)
-    info.graph_.add_to_collection(transformed_name, elem_)
+
+    target_graph = info.graph_ # type: Graph
+    if isinstance(elem_, (Node, Tensor, Variable)):
+      elem_.add_to_collection(transformed_name)
+    else:
+      raise NotImplementedError("Don't know how to add name '{}' to target "
```

Comment:
This is minor, but error might make be more clear with something like "Unable to add '{}' to target " + ...

Reply:
Good point, added a more complete explanation.